### ISSUE TITLE
[a11y] Drawer menu button is not reachable by screen readers

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/website#1160]
On iOS or Android, turn on TalkBack/VoiceOver. When moving focus through the app bar, the hamburger is skipped, and as a result the side drawer cannot be opened.

**Comment by Sfshaza** [comment]
@jonahwilliams, can you open this bug on flutter/flutter/issues ? Unless you believe it's a documentation issue, but it doesn't sound like it.

**Comment by jonahwilliams** [comment]
ahh woops, closing
